client/jetpack-connect/remote-credentials.js

```diff
@@ -245,12 +241,6 @@ export class OrgCredentialsForm extends Component {
 						) }
 					</div>
 				</div>
-				<div className="jetpack-connect__note">
```

Comment:
Same as above - this `className` is no longer in use and has styles that we can remove as part of the PR.